r124/u_inref/ydb312_gtm8182a.csh

```diff
@@ -0,0 +1,64 @@
+#!/usr/local/bin/tcsh -f
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+# Set white box testing environment to avoid assert failures along with REPLMULTINSTUPDATE error
```

Comment:
Lines 14-16 seem unnecessary. They are needed only if the test is exercising a REPLMULTINSTUPDATE error. Please fix this in all gtm8182* subtests as applicable.